x-pack/plugins/enterprise_search/public/applications/shared/error_state/error_state_prompt.tsx

```diff
@@ -54,6 +56,26 @@ export const ErrorStatePrompt: React.FC = () => {
                 defaultMessage="Confirm that the Enterprise Search server is responsive."
               />
             </li>
+            <li>
+              <FormattedMessage
+                id="xpack.enterpriseSearch.errorConnectingState.troubleshootAuth"
+                defaultMessage="Check your user authentication:"
+              />
+              <ul>
+                <li>
+                  <FormattedMessage
+                    id="xpack.enterpriseSearch.errorConnectingState.troubleshootAuthNative"
+                    defaultMessage="Are you using Elasticsearch Native auth?"
+                  />
+                </li>
+                <li>
+                  <FormattedMessage
+                    id="xpack.enterpriseSearch.errorConnectingState.troubleshootAuthSAML"
+                    defaultMessage="If you’re on SSO or SAML, is your SAML realm also set up on Enterprise Search?"
+                  />
```

Comment:
@gchaps / @chriscressman - apologies for the last minute notice, but is there any chance we can get a quick copy check/pass on this new text? (Feel free to view the screenshot if that's easier!)

Reply:
Sure thing @constancecchen . Here are some suggested changes. I'm trying to be a bit more explicit about what they must do to fix the issue:

> Confirm your user authentication:
> * You must authenticate using Elasticsearch Native auth or SSO/SAML
> * If using SSO/SAML, you must set up your SAML realm on Enterprise Search

Reply:
> If using SSO/SAML, you must set up your SAML realm on Enterprise Search

I know this is a bit more passive, but do we want to consider changing this to:

> If using SSO/SAML, your SAML realm must be set up on Enterprise Search

The only reason why I ask this is because on Elastic Cloud, SSO is currently not set upon Enterprise Search and there is absolutely nothing the admin/operator can do about it until Cloud itself ships that functionality in 6 weeks.

I know this is a pretty special case though, and do think the active voice is generally better/applies to other SAML setups - just wanted to throw that context out there!

Reply:
Yes, your version looks good to me. I can see how in that scenario (and possibly others) the person setting up Enterprise Search in Kibana doesn't have control over the SAML setup.